Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/dao/SuitDAOImpl.java

```diff
@@ -4,30 +4,45 @@
 import com.epam.test_generator.entities.Suit;
 import org.springframework.stereotype.Repository;
 
+import java.util.ArrayList;
 import java.util.List;
 
 @Repository
 public class SuitDAOImpl implements EntitiesDAO<Suit>{
```

Comment:
I suggest to mark in the name of the class that it is temporary solution ie SuitDaoMockImlp, and then later autowire it with name and qualifier.